app/src/util/types.ts

```diff
@@ -159,3 +159,9 @@ export interface MarketMakerData {
   userEarnings: BigNumber
   userPoolShares: BigNumber
 }
+
+export enum Ternary {
```

Comment:
Added this enum for the scenarios where we have a flag whose value can be unknown.

We were using `boolean | null`, but I find it less readable and more error-prone. That being said, using this type has the risk of doing something like `if (flag)` that will always be true. This can be fixed with a linter rule, but it's very slow, so maybe we should run that rule only in the CI.